### ISSUE TITLE
MinGW builds in most recent MSVC 2019 image fail

### ISSUE THREAD

**Issue by randombit** [appveyor/ci#3392]
I'm seeing a problem with my MinGW CI builds where trying to invoke the binary results in error

"Command exited with code -1073741511"

which AIUI is a "File not found" or "DLL not found" error.

The build fails with the current Visual C++ 2019 image but succeeds using the previous Visual C++ 2019 image (and has been working fine since November of last year when first introduced into our CI).

Failing (Current image): https://ci.appveyor.com/project/randombit/botan/builds/32370711
Working (Previous image): https://ci.appveyor.com/project/randombit/botan/builds/32370802

These builds are identical source except for the choice of image. Cloning master of https://github.com/randombit/botan should be able to reproduce. AppVeyor configuration is at https://github.com/randombit/botan/blob/master/src/scripts/ci/appveyor.yml

An interesting thing here is the failing image reports the GCC as

> gcc version 9.3.0 (Rev2, Built by MSYS2 project)

but working (in Previous image) reports

> gcc version 9.3.0 (Rev1, Built by MSYS2 project)

So GCC has changed, but there is no mention of MinGW changes for the April 9 image in https://www.appveyor.com/updates/

**Comment by FeodorFitsner** [comment]
That's true, every time a new image is backed it pulls the latest MSYS2. This issue looks more like upstream issue, rather than platform issue. I think the best way is to report MSYS team about the issue.

**Comment by Biswa96** [comment]
If possible try to updates the packages first using `pacman -Syyu` command then build the project. Use `mingw-w64-x86_64-ntldd-git` package to detect the dependent DLLs with which you can get the missing dependent packages. Also Appveyor may need to update MSYS2 installation as there are some recent core changes in MSYS2.
